src/Symfony/Component/Serializer/Normalizer/DataUriNormalizer.php

```diff
@@ -89,7 +89,7 @@ public function supportsNormalization(mixed $data, string $format = null, array
      */
     public function denormalize(mixed $data, string $type, string $format = null, array $context = []): \SplFileInfo
     {
-        if (null === $data || !preg_match('/^data:([a-z0-9][a-z0-9\!\#\$\&\-\^\_\+\.]{0,126}\/[a-z0-9][a-z0-9\!\#\$\&\-\^\_\+\.]{0,126}(;[a-z0-9\-]+\=[a-z0-9\-]+)?)?(;base64)?,[a-z0-9\!\$\&\\\'\,\(\)\*\+\,\;\=\-\.\_\~\:\@\/\?\%\s]*\s*$/i', $data)) {
+        if (null === $data || !preg_match('/^data:([a-z0-9][a-z0-9\!\#\$\&\-\^\_\+\.]{0,126}\/[a-z0-9][a-z0-9\!\#\$\&\-\^\_\+\.]{0,126}(;[a-z0-9\-]+\=[a-z0-9\-]+)?)?(;base64)?,[a-z0-9\!\$\&\\\'\,\(\)\*\+\;\=\-\.\_\~\:\@\/\?\%\s]*\s*$/i', $data)) {
```

Comment:
Only Jesus can review this. :smiling_face_with_tear: Looks like it should be in separate PR and possibly with test case for the current issue.

Reply:
This is easier than it looks; it's just that the `\,` was included twice in the regexp.